src/web/nextui/src/app/components/ApiSettingsModal.tsx

```diff
@@ -0,0 +1,87 @@
+import { useState, useEffect } from 'react';
```

Comment:
Settings modal that only shows in Dev or running from the CLI. This lets you set the Remote API URL and Share URLs if you want.

This does let you run stuff locally and then share to any hosted instance.